backend/tracim_backend/lib/utils/utils.py

```diff
@@ -492,3 +495,13 @@ def get_build_version(path: str) -> str:
     except StopIteration:
         # INFO - G.M - 2020-01-13 - return the 10 first letter of current commit hash
         return repo.head.object.hexsha[:10]
+
+
+def validate_page_token(page_token: str):
+    # INFO - G.M - 2020-07-23 - Explicitly catch error for unvalid bookmark
+    # as unserialize_bookmark method of sqlakeyset doesn't return explicit exception
+    # see https://github.com/djrobstep/sqlakeyset/issues/34
```

Comment:
```suggestion
    # INFO - G.M - 2020-07-23 - Because they lack an explicit message, we catch exceptions from
    # the unserialize_bookmark method of sqlakeyset and re-raise them with an explicit message.
    # See https://github.com/djrobstep/sqlakeyset/issues/34
```